spec/draft-barnes-cfrg-4q.md

```diff
@@ -579,28 +583,27 @@ Convert T[7] to R2
 
 ### Scalar Decomposition and Recoding
 
-This stage has two parts. The first is to decompose the scalar into
-four small integers, the second is to encode these integers into a
-form that can be used to efficiently compute the scalar
-multiplication.
+This stage has two parts. The first one consists in decomposing the scalar into
+four 64-bit integers, and the second one consists in recoding these integers into a
+form that can be used to efficiently and securely compute the scalar multiplication.
 
-This decomposition uses four vectors with 64 bit entries namely b1,
-b2, b3, b4.  In addition we have l1, l2, l3, l4 which are long
-integers used to implement rounding.
+The decomposition step uses four fixed vectors called b1, b2, b3, b4, with 64 bit entries each.
+In addition, we have l1, l2, l3, l4, which are long integers used to implement rounding.
```

Comment:
"Long integer" is not defined.  Let's just be specific and say "192-bit integers"?